src/main/resources/config/application.yml

```diff
@@ -34,10 +34,34 @@ vnfmdriver:
     lcmOpOccPollingTopic: lcm_op_occ_polling_requests
     lifecycleResponsesTopic: lm_vnfc_lifecycle_execution_events
   packageManagement:
+    enabled: true
     imageArtifactFilter: Files/Images
     # url should be in the form 'http://hostname/path/{vnfPackageId}'
     packageRepositoryUrl:  
   grant:
     automatic: true
     provider:
       url:
+
+ribbon:
+  eureka:
+    enabled: true
+  # Set timeout on Zuul proxy to 20 seconds
+  ReadTimeout: 20000
```

Comment:
Might need to increase this or disable the Read timeout since we could be downloading large (multiple Gb files)

Reply:
ReadTimeout now set to 30mins